tests/test_basic/results.py

```diff
@@ -2,7 +2,7 @@
 
 import sys
 
-sys.path.insert(0, '../../src/utils')
+sys.path.insert(0, '../..')
```

Comment:
I collocated the `sys.path.insert(...)` line with the imports so that it is now more obvious what the purpose is.